fortuna/prob_model/posterior/normalizing_flow/advi/advi_posterior.py

```diff
@@ -63,6 +67,7 @@ def fit(
         self,
         train_data_loader: DataLoader,
         val_data_loader: Optional[DataLoader] = None,
+        map_fit_config: Optional[FitConfig] = None,
```

Comment:
I'd rather have a user explicitly train a Model using MAP and then training with ADVi on top by either passing the MAP state or the checkpoint. I think that having `map_fit_config` and training MAP within ADVI adds an unnecessary overhead.